### ISSUE TITLE
Missing dependency

### ISSUE THREAD

**Issue by ItzSwirlz** [linuxmint/folder-color-switcher#23]
When I built and installed folder-color-switcher, it did nothing on my Cinnamon (this is on Ubuntu, not Mint). It didn't do anything until I looked at the code and saw there was some nemo-python scripts for it. The issue is clearly, the package dependencies are only Python's Depends and not including nemo-python which would work in my case.

Since you have caja-python and that stuff, it might make sense to possibly create packages for the desktops seperately-just add the lines to the control file and this issue can be fixed.

TL;DR: Python-Nemo isn't a dependency, which it is-and since Caja-Python exists there should be two packages: folder-color-switcher-nemo and folder-color-switcher-caja.

**Comment by ItzSwirlz** [comment]
Trying to do what I can in order to get this into Debian/Ubuntu repos. This is a wonderful, wonderful package.

**Comment by ItzSwirlz** [comment]
STATUS UPDATE: I built and installed python-nemo-made no difference. Installed all nemo packages, nemo-* and nothing happened. Clearly a) package isnt working (unlikely) or b) it is a mint only package (possible) or c) the package is missing a dependency (most likely). Please investigate.

**Comment by leigh123linux** [comment]
You need to use a mint-*-theme for folder-color-switcher to function

**Comment by ItzSwirlz** [comment]
I will open a pull request then.

**Comment by ItzSwirlz** [comment]
But still it is a bit of a shame-there isn't any folder color switcher for cinnamon-you can only use folder-color in GNOME desktop then go over to cinnamon

**Comment by ItzSwirlz** [comment]
Probably fixed by now.
